shoppingfeed.php

```diff
@@ -629,31 +629,37 @@ public function mapPrestashopProduct($sfProductReference, $id_shop, ...$argument
      */
     public function mapProductPrice(ShoppingfeedProduct $sfProduct, $id_shop, $arguments = [])
     {
-        $cloneContext = Context::getContext()->cloneContext();
-        $cloneContext->shop = new Shop($id_shop);
-
         $specific_price_output = null;
         Product::flushPriceCache();
-        $price = Product::getPriceStatic(
-            $sfProduct->id_product, // id_product
-            true, // usetax
+
+        // Tax depends on a country. We use country configured as default one for shop.
+        $id_country = Configuration::get('PS_COUNTRY_DEFAULT', null, null, $id_shop);
+        $id_currency = Validate::isLoadedObject($this->context->currency) ? (int) $this->context->currency->id : (int) Configuration::get('PS_CURRENCY_DEFAULT', null, null, $id_shop);
```

Comment:
I always use this configuration Configuration::get('PS_CURRENCY_DEFAULT', null, null, $id_shop)
If the Context is set for an other Shop with a different currency, I don't know what's happend..